app/services/responsibility_service.rb

```diff
@@ -24,6 +24,12 @@ def self.nps_calculation(offender)
   end
 
   def self.assign_responsible?(offender)
```

Comment:
I cannot see this being called anywhere 🤔

Reply:
It's called at https://github.com/ministryofjustice/offender-management-allocation-manager/pull/314/files#diff-0f5c3ebaf1f0ae922a6ea7b08b835923R15 in the fold above